src/_data/snippets/min_tilt_version.yml

```diff
@@ -1,7 +1,6 @@
 ---
 title: Enforce a minimum Tilt version
-description: |
-  Require a minimum version of Tilt to ensure all expected features are available.
+description: Require a minimum version of Tilt for feature availability
```

Comment:
I was also wondering whether this snippet should just use
```
version_settings(check_updates=True, constraint='>=0.23.7')
```

Reply:
Oh yeah, that's way better! I forgot about that built-in 😬